python/pyspark/testing/connectutils.py

```diff
@@ -97,6 +97,7 @@ def _session_sql(cls, query: str) -> "DataFrame":
         return DataFrame.withPlan(SQL(query), cls.connect)  # type: ignore
 
     if have_pandas:
+
```

Comment:
Sorry, but I piggy-back this python-style fix. This style issue is introduced in the previous commit separately today.
```
$ dev/reformat-python
reformatted python/pyspark/testing/connectutils.py

All done! ✨ 🍰 ✨
1 file reformatted, 391 files left unchanged.
```

Reply:
Thanks

Reply:
cc @zhengruifeng